engine/src/main/java/org/hibernate/validator/internal/xml/ValidationXmlParser.java

```diff
@@ -103,6 +103,7 @@ public final BootstrapConfiguration parseValidationXml() {
 			ValidationConfigStaxBuilder validationConfigStaxBuilder = new ValidationConfigStaxBuilder( xmlEventReader );
 
 			xmlEventReader.close();
+			in.reset();
```

Comment:
Let's move this change to the first commit too. Usually, I just apply the change there when editing the commit in the rebase and usually git ignores the change afterwards when continuing the rebase.